plugins/quay/src/hooks/quay.tsx

```diff
@@ -64,6 +68,7 @@ export const useTags = (organization: string, repository: string) => {
       const hashFunc = tag.manifest_digest.substring(0, 6);
       const shortHash = tag.manifest_digest.substring(7, 19);
       return {
+        id: Math.floor(Math.random() * 99999999),
```

Comment:
can we use `tag.manifest_digest` here?

Reply:
used tag manifest and tag name to create a unique id